examples/VAE_example/README.md

```diff
@@ -0,0 +1,39 @@
+# Federated Variational Auto-encoder Example
+This is an example implementation of training a Variational Auto_encoder model on the MNIST dataset. In this example, the client is a basic client, but it can instead inherit from any of the available client classes. The server uses Federated Averaging to aggregate the CVAE model weights. The structure of the encoder and decoder models are defined in 'models.py'.
+
+## Running the Example
+In order to run the example, first ensure you have the virtual env of your choice activated and run
+```
+pip install --upgrade pip
```

Comment:
Small note: In the most recently merge PR, Marcelo updated the instructions for installation in the examples to reference the README, you probably just copy and past that to replace these old installation instructions

Reply:
Right, I should do that!